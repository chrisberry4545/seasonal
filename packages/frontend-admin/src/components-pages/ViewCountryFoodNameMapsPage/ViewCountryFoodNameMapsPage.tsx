import React, { FC } from 'react';
import { FullListCountryFoodNameMaps } from '../../components-main';

export const ViewCountryFoodNameMapsPage: FC<{}> = () => <FullListCountryFoodNameMaps />;
