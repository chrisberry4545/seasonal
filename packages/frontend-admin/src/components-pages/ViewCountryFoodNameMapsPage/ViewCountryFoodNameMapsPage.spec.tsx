import React from 'react';
import { ViewCountryFoodNameMapsPage } from './ViewCountryFoodNameMapsPage';
import { shallow, ShallowWrapper } from 'enzyme';

jest.mock('../../components-main', () => ({
  FullListCountryFoodNameMaps: () => 'FullListCountryFoodNameMaps'
}));

describe('<ViewCountryFoodNameMapsPage />', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() =>
    wrapper = shallow(
      <ViewCountryFoodNameMapsPage />
    )
  );

  test('renders correctly', () => expect(wrapper).toMatchSnapshot());

});
