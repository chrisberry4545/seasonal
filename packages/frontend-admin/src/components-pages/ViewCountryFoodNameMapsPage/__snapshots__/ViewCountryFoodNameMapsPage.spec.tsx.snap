// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ViewCountryFoodNameMapsPage /> renders correctly 1`] = `<FullListCountryFoodNameMaps />`;
